ghost/i18n/locales/de/portal.json

```diff
@@ -25,12 +25,12 @@
     "Cancel subscription": "Abonnement kündigen",
     "Cancellation reason": "Kündigungsgrund",
     "Change": "Ändern",
-    "Change plan": "",
+    "Change plan": "Tarif änder",
```

Comment:
@ronaldlangeveld this looks like a typo — should be `Tarif ändern` I'd say (except there's some context that I don't have that adds an `n`).